screens/DaterDashboard.tsx

```
@@ -0,0 +1,211 @@
+    return (
+        <ScrollView style={commonStyles.container} contentContainerStyle={commonStyles.contentContainer}>
+            
+            <AppBar />
+
+            {/* User Header */}
+            <ThemedView style={commonStyles.userHeader}>
+                <ThemedImage 
+                    source={getProfileImage(dater.image)} 
+                    style={commonStyles.userImageLarge}
+                />
+                <ThemedText style={[commonStyles.title, { textAlign: 'center' }]}>{dater.name}'s Dashboard</ThemedText>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)